### DISCUSSION TITLE
HTTP -> HTTPS redirect seem to be buggy

### DISCUSSION THREAD

**Discussion by raphaelbernhart** [coollabsio/coolify#1604] [Q&A]
While deploying an application I encountered weird behaviour when enabling the "Force HTTPS" flag.

I have a domainname and I want `.www.DOMAIN` and `DOMAIN` to be redirected to https. Unfortunately I could not figure out how to do it. In the Domain Input I entered all FQDN with http and https prefix ([image](https://github.com/coollabsio/coolify/assets/48283236/eeba03bd-fcf4-45f1-be31-04acc26526ab)). The value I entered was `https://${DOMAIN},https://www.${DOMAIN},http://${DOMAIN},http://www.${DOMAIN}` ([image](https://github.com/coollabsio/coolify/assets/48283236/4c2c6ced-fb75-4a7f-9c21-d20cca8d1a80)).

When resolving `https://${DOMAIN}` and `https://www.${DOMAIN}` everything is working fine but when resolving `http://${DOMAIN}` and `http://www.${DOMAIN}` I get redirected to the 404 redirect url ([image](https://github.com/coollabsio/coolify/assets/48283236/e1db52f2-a125-4f7c-aafe-97691aca8264)).

curl to `http://${DOMAIN}` [image](https://github.com/coollabsio/coolify/assets/48283236/61a5e154-f3c4-49a8-9baf-bdbe464bf738) (⛔ - should get redirected to https)
curl to `https://${DOMAIN}`[image](https://github.com/coollabsio/coolify/assets/48283236/e124bda0-491f-4701-9ec9-0126f3e90732) (✅)

Am I configuring it wrong? Is this just buggy behaviour?

**Comment by andrasbacsai** [comment]
This has been solved in the next version (beta.184 - not released yet). After updating, please regenerate the labels.

**Reply by raphaelbernhart** [reply]
Thanks, seems to work now :)
